badgerdb/badgerdb.go

```diff
@@ -127,7 +167,14 @@ func NewStore(options Options) (Store, error) {
 
```

Comment:
In the `NewStore` function I used to explicitly set the default values if a value was set to a zero value which _probably_ wasn't intended to be used as an actual value. It's clear for `nil`, but for empty strings often times as well.
So in this case, what do you think about the zero values of `MaxCacheSize` (maybe set to 1 GB) and `EncryptionKeyRotationDuration`? The zero values are useful here, right?